src/docs/docs.yaml

```diff
@@ -197,6 +287,96 @@ definitions:
           is_verified:
             type: boolean
             default: false
+          role:
+            type: string
+            default: requester
+          manager_id:
+            type: integer
+            nullable: true
+          gender:
+            type: string
+            nullable: true
+          date_of_birth:
+            type: string
+            format: date
+            nullable: true
+          department:
+            type: string
+            nullable: true
+          preferred_language:
+            type: string
+            nullable: true
+          residential_address:
+            type: string
+            nullable: true
+          preferred_currency:
+            type: string
+            nullable: true
+          createdAt:
+            type: string
+            format: date-time
+          updatedAt:
+            type: string
+            format: date-time
+  UsersResponse:
+    properties:
+      status:
+        type: string
+      data:
+        type: object
+        properties:
+          id:
+            type: integer
+          email:
+            type: string
+            format: email
+          first_name:
+            type: string
+          last_name:
+            type: string
+          is_admin:
+            type: boolean
+            default: false
+          is_verified:
+            type: boolean
+            default: false
+          role:
+            type: string
+            default: requester
+          manager_id:
+            type: integer
+            nullable: true
+          gender:
+            type: string
+            nullable: true
+          date_of_birth:
+            type: string
+            format: date
+            nullable: true
+          department:
```

Comment:
Please remove `manager_id` and `department`

Reply:
Alright

Reply:
Do I also remove it from the model and migrations?

Reply:
Yes, we are replacing that with `role_id`

Reply:
Ok